test/integration/backward_compatible/ClientHotRestartEventTest.js

```diff
@@ -57,7 +57,9 @@ describe('ClientHotRestartEventTest', function () {
     });
```

Comment:
I want to add client = undefined here to be sure

Reply:
because first test may set client, the second might skip. in this case client.shutdown will be called again on the same client

Reply:
yes, I was about the say the same thing